Core/Object Arts/Dolphin/Base/PackageManager.cls

```diff
@@ -510,7 +510,10 @@ install: aString
 	[newPackages do: 
 			[:each | 
 			self addPackage: each.
-			[self basicInstall: each] ifCurtailed: [self basicUninstall: each].
+			[
+				self basicInstall: each.
+				self trigger: #packageInstalled with: each.
```

Comment:
I did also wonder if this was the right place to trigger the event, given that only the basic installation has been performed and the manager is not yet observing the package. However it makes since in that if an event handler throws an exception, the package will be uninstalled by the curtailment handler. Therefore I think this is right.